setup.py

```diff
@@ -9,7 +9,7 @@ def read(fname):
 
 setup(
     name = "django-form-designer",
-    version = "0.8.0.5",
+    version = "0.9.0",
```

Comment:
Don't know if it's Github again which does not show the commits in ancestry order, but shouldn't this commit be the last one? (Github shows this as the second last.)